pkg/util/util_test.go

```diff
@@ -208,11 +209,18 @@ func TestCloneRepo(t *testing.T) {
 			repo:      "https://github.com/devfile-samples/devfile-sample-java-springboot-basic",
 			wantErr:   false,
 		},
+		{
```

Comment:
I assume there is no straightforward way to test a happy path

Reply:
Not without:
1) A real git token and real private repository to use
or
2) Mocking go-git's CloneRepo

2 might be worth exploring in the future, especially if we need to do anything else with go-git, but I'd say it's outside of the scope of this PR.